modules/core/src/testFixtures/java/org/apache/ignite/internal/testframework/IgniteTestUtils.java

```diff
@@ -891,4 +894,40 @@ public boolean matches(Object o) {
             return predicate.test((DataT) o);
         }
     }
+
+    /**
+     * Creates a matcher that tests if a CompletableFuture has completed successfully.
+     *
+     * @param <T> the type of the CompletableFuture
+     * @return a matcher for a successfully completed CompletableFuture
+     */
+    public static <T> Matcher<CompletableFuture<T>> completedSuccessfully() {
```

Comment:
How about moving this to `CompletableFutureMatcher`?

Reply:
I moved it to a separate class